static/js/index.js

```diff
@@ -3055,6 +3061,58 @@ function getPokestopPopupContent (pokestop) {
     return content;
 }
 
+function getPossibleInvasionRewards (pokestop) {
+    let item = gruntTypes[pokestop.grunt_type];
+    let content = '';
+	content +=
+	//'<input class="button" name="button" type="button" onclick="showHideGruntEncounter()" value="Show / Hide Possible Rewards" style="margin-top:2px; outline:none; font-size:9pt">' +
+    //'<div class="grunt-encounter-wrapper text-center" style="display:none; background-color:#1f1f1f; border-radius:10px; border:1px solid black;">'
+    '<div class="grunt-encounter-wrapper text-center" style="background-color:#1c1c1c; border-radius:10px; border:1px solid black;">'
+	if (item['second_reward'] === 'false') {
+		content += `<div>100% Encounter Chance:<br>`;
+		item['encounters']['first'].forEach(function (data) {
+            content += `
+            <img src="${availableIconStyles[selectedIconStyle].path}/${getPokemonIcon(data)}.png" class="invasion-reward" />
+            <img src="img/misc/shadow.png" class="invasion-reward-shadow m-1"/>
+            `;
+		});
+        content += `</div>
+        </div>`;
+	} else if (item['second_reward'] === 'true') {
+		content += `<div>85% Encounter Chance:<br>`;
+		item['encounters']['first'].forEach(function (data) {
+            content += `
+            <img src="${availableIconStyles[selectedIconStyle].path}/${getPokemonIcon(data)}.png" class="invasion-reward" />
+            <img src="img/misc/shadow.png" class="invasion-reward-shadow m-1"/>
+            `;
+		});
+		content += `</div>
+		<div class="m-1">15% Encounter Chance:<br>`;
+		item['encounters']['second'].forEach(function (data) {
+            content += `
+            <img src="${availableIconStyles[selectedIconStyle].path}/${getPokemonIcon(data)}.png" class="invasion-reward" />
+            <img src="img/misc/shadow.png" class="invasion-reward-shadow m-1"/>
+            `;
+		});
+        content += `
+        </div>
+    </div>`;
+	}
+    return content;
+}
+
+function showHideGruntEncounter() {
+    var x = document.getElementsByClassName('grunt-encounter-wrapper')
+    var i
```

Comment:
Why the inconsistency of `;`'s and `var`/`let`?

Reply:
Oops